metricbeat/module/kubernetes/_meta/kibana/7/dashboard/Metricbeat-kubernetes-scheduler.json

```
@@ -0,0 +1,847 @@
+            "panelRefName": "panel_3",
+            "version": "7.2.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 11,
+              "i": "5",
+              "w": 23,
+              "x": 25,
+              "y": 14
+            },
+            "panelIndex": "5",
+            "panelRefName": "panel_4",
+            "version": "7.2.0"
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Metricbeat Kubernetes] Scheduler",
+        "version": 1
+      },
+      "id": "f5ab5510-9c94-11e9-94fd-c91206cd5249",
+      "migrationVersion": {
+        "dashboard": "7.0.0"
+      },
+      "references": [
+        {
+          "id": "metricbeat-*",
+          "name": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
+          "type": "index-pattern"
+        },
+        {
+          "id": "bcb194a0-9bf8-11e9-9dc8-fd27291d427f",
+          "name": "panel_0",
+          "type": "visualization"
+        },
+        {
+          "id": "826d80c0-9c97-11e9-94fd-c91206cd5249",
+          "name": "panel_1",
+          "type": "visualization"
+        },
+        {
+          "id": "15bd4420-9c9b-11e9-94fd-c91206cd5249",
+          "name": "panel_2",
+          "type": "visualization"
+        },
+        {
+          "id": "d9fc1b80-9c9c-11e9-94fd-c91206cd5249",
+          "name": "panel_3",
+          "type": "visualization"
+        },
+        {
+          "id": "95595810-9ca8-11e9-94fd-c91206cd5249",
+          "name": "panel_4",
+          "type": "visualization"
+        }
+      ],
+      "type": "dashboard",
+      "updated_at": "2019-07-02T09:12:58.678Z",
+      "version": "WzY5MSwxXQ=="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [
+              {
+                "$state": {
+                  "store": "appState"
+                },
+                "meta": {
+                  "alias": null,
+                  "disabled": false,
+                  "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
+                  "key": "event.dataset",
+                  "negate": false,
+                  "params": {
+                    "query": "kubernetes.controllermanager"
+                  },
+                  "type": "phrase",
+                  "value": "kubernetes.controllermanager"
+                },
+                "query": {
+                  "match": {
+                    "event.dataset": {
+                      "query": "kubernetes.controllermanager",
+                      "type": "phrase"
+                    }
+                  }
+                }
+              }
+            ],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "Kubernetes - Host selector controller manager",
```

We follow this schema for visualization titles:
```suggestion
        "title": "Host selector controller manager [Metricbeat Kubernetes]",
```

Once a visualization is placed in a dashboard, its title can be customized, and then there is no need to mention the module as the dasboard is already titled with that. So in the dashboard this visualization could be titled `Host selector controller manager`.